Hive.NET.Core/Models/Enums/BeeWorkItemStatus.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Hive.NET.Core.Models.Enums;
+
+public enum BeeWorkItemStatus
+{
+    Waiting,
```

Comment:
Maybe add explicit values to enum. Would be safer if someone would like to expand it

Reply:
done bruh